src/python/pants/backend/typescript/tsconfig.py

```diff
@@ -77,8 +78,39 @@ async def _read_parent_config(
     )
 
 
+def _clean_tsconfig_contents(content: str) -> str:
+    """The tsconfig.json uses a format similar to JSON, but there are some important differences:
```

Comment:
I think it's "JSON with comments": https://code.visualstudio.com/docs/languages/json#_json-with-comments

Maybe worth linking to that?

Reply:
Oh that what I thought it is first, but the trailing commas made me hesitant. Apparently the trailing commas are allowed, too. Extended the docstring!